packages/web-components/fast-components/src/radio-group/fixtures/base.html

```diff
@@ -0,0 +1,49 @@
+<fast-design-system-provider use-defaults>
+    <h1>Radio Group</h1>
+
+    <h4>Defaults</h4>
+    <div>
+        <fast-radio-group name="numbers">
+            <fast-radio value="one">One</fast-radio>
+            <fast-radio value="two">Two</fast-radio>
+        </fast-radio-group>
+    </div>
+
+    <div style="display: flex; flex-direction: column;margin-top: 12px;">
+        <label>Single radio (no group)</label>
+        <fast-radio value="single-life">Single life</fast-radio>
+    </div>
+
+    <div style="display: flex; flex-direction: column;margin-top: 12px;">
+        <label id="label1">Outside label</label>
+        <fast-radio-group aria-labelledby="label1" name="fruits">
+            <fast-radio value="apples">Apples</fast-radio>
+            <fast-radio value="oranges">Oranges</fast-radio>
+            <fast-radio value="bananas">Bananas</fast-radio>
+        </fast-radio-group>
+    </div>
+
+    <div style="display: flex; flex-direction: column;margin-top: 12px;">
+        <label id="label2">Disabled radio group</label>
+        <fast-radio-group disabled aria-labelledby="label2" name="cars">
+            <fast-radio value="lambo">Lamborghini</fast-radio>
+            <fast-radio value="ferari">Ferari</fast-radio>
+        </fast-radio-group>
+    </div>
+
+    <div style="display: flex; flex-direction: column;margin-top: 12px;">
+        <label id="label3">readonly radio group</label>
+        <fast-radio-group readonly aria-labelledby="label3" name="office">
+            <fast-radio value="word">Word</fast-radio>
+            <fast-radio value="excel">Excel</fast-radio>
+        </fast-radio-group>
+    </div>
+
+    <div style="display: flex; flex-direction: column;margin-top: 12px;">
+        <label id="label4">Preset selected-value</label>
```

Comment:
Should we surface a label slot?

Reply:
+1

Reply:
I'm adding a slot named "label" and adding back in a control div so I can control the vertical stacking of the label if slotted and the radios, it also means I can easily add vertical and horizontal classes to accommodate orientation attribute for radio group